hazelcast/proxy/vector_collection.py

```diff
@@ -91,10 +198,28 @@ def search_near_vector(
         *,
         include_value: bool = False,
         include_vectors: bool = False,
-        limit: int,
+        limit: int = 10,
         hints: Dict[str, str] = None
     ) -> Future[List[SearchResult]]:
+        """Returns the Documents closest to the given vector.
+
+        The search is performed using distance metric set when
+        creating the vector index.
+
+        Args:
+            vector: The vector to be used as the reference.
+                It must have the same dimension as specified when creating the vector index.
+            include_value: Return value attached to the Document.
+            include_vectors: Return vectors attached to the Document.
```

Comment:
```suggestion
            include_value: Whether to return value attached to the Document.
            include_vectors: Whether to return vectors attached to the Document.
```
I think better, not strong

Reply:
I think both versions are fine.